styles/style.css

```diff
@@ -339,7 +339,7 @@ body {
 }
 
```

Comment:
- [ ] Kindly fix your styling on the popup windows, I'm running it on my computer and it's seems that there's some images missing on the project

![image](https://user-images.githubusercontent.com/93630700/211115088-1c4a3a36-329e-47a3-b9d4-4589dda59d67.png)